hp_list/main.c

```diff
@@ -195,7 +325,7 @@ list_node_t *list_node_new(list_key_t key)
     list_node_t *node = aligned_alloc(128, sizeof(*node));
     assert(node);
     *node = (list_node_t){.magic = LIST_MAGIC, .key = key};
-    (void) atomic_fetch_add(&inserts, 1);
+    TRACE_INSERTS;
```

Comment:
Don't do this!
Instead, you should rewrite with the wrapper:
```diff
-    (void) atomic_fetch_add(&inserts, 1);
+    ATOMIC_FETCH_ADD(&inserts, 1, INS);
```

Here `INS` denotes the operations of insertions. Similarly, you can use `DEL` for tracing deletions.

Reply:
Of course, you can think of the 1-to-1 mapping from `__atomic_` builtins to `ATOMIC_` macros without additional parameters.